diff --git a/index.d.ts b/index.d.ts
index ba18b251569f034540d1168c15b4d5768609216b..e4c15170c9628c676f1b35b3863050354d72fde6 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -362,8 +362,9 @@ declare module "react-calendar-timeline" {
     export const defaultTimeSteps: TimelineTimeSteps;
     export const defaultHeaderFormats: LabelFormat;
 
-    export default class ReactCalendarTimeline<
-        CustomItem extends TimelineItemBase<any> = TimelineItemBase<number>,
-        CustomGroup extends TimelineGroupBase = TimelineGroupBase,
-    > extends React.Component<ReactCalendarTimelineProps<CustomItem, CustomGroup>> {}
+    declare const ReactCalendarTimeline: <
+    CustomItem extends TimelineItemBase<any> = TimelineItemBase<number>,
+    CustomGroup extends TimelineGroupBase = TimelineGroupBase,
+>(props: ReactCalendarTimelineProps<CustomItem, CustomGroup>) => React.ReactElement;
+export default ReactCalendarTimeline;
 }
